bindings/rust/s2n-tls/src/security.rs

```diff
@@ -108,6 +108,9 @@ pub const DEFAULT_TLS13: Policy = policy!("default_tls13");
 #[cfg(feature = "pq")]
 pub const TESTING_PQ: Policy = policy!("PQ-TLS-1-0-2021-05-26");
 
+#[cfg(any(feature = "unstable-testing", test))]
+pub const TESTING_TLS12: Policy = policy!("20240501");
+
```

Comment:
Nit: might want to put this above the pq policies instead of inserting it between them